test/src/main/scala/scala/test/junit/specs2/Specs2Tests.scala

```diff
@@ -0,0 +1,16 @@
+package scala.test.junit.specs2
+
+import org.specs2.mutable.SpecWithJUnit
+import scala.test.junit.support.JUnitCompileTimeDep
+
+class JunitSpecs2Test extends SpecWithJUnit {
+
+  "specs2 tests" should {
+    "run smoothly in bazel" >> {
+      JUnitCompileTimeDep.hello
+      success
+    }
```

Comment:
can we add a negative test in the failing tests path? We should show that something that should fail does.